eng/common/yaml/create-tags-and-git-release.yml

```diff
@@ -0,0 +1,18 @@
+parameters:
```

Comment:
I would put this in a similar structure as our other pipelines stuff. So `eng/common/pipelines/templates/steps/create-and-git-release.yml`